x-pack/plugins/cases/server/client/attachments/add.ts

```diff
@@ -184,12 +187,12 @@ const addGeneratedAlerts = async ({
         comment: query,
         status: subCase.attributes.status,
       });
-      await casesClient.updateAlertsStatus({
+      await casesClientInternal.alerts.updateStatus({
```

Comment:
yes that 's what I was expected nice!, I am sure I am missed something above that ;)

Reply:
I haven't update the tests yet :)